### ISSUE TITLE
Update C779 from Cert to OA class

### ISSUE THREAD

**Issue by Krautpaddy** [Krautpaddy/myBSCS-Classes-Notes#1]
C779 has since been changed from a certification course granting a CIW Site Development Associate certification to a regular Objective Assessment. I need to update the course page with that, find any write-ups with updated information, and find out the new OA competencies and %s.

**Comment by Krautpaddy** [comment]
I've updated the C779 page, but I still need to find more information on the OA breakdown, competencies, and find more write-ups from students who have taken the new OA on what helped them/what should be added to the course page.

**Comment by todgru** [comment]
@Krautpaddy I just started the course last night. I created a pull request with the breakdown of the class. I added a new Quizlet covering the vocabulary section from the `cmweb` user. https://github.com/Krautpaddy/myBSCS-Classes-Notes/pull/2

Also, I used Github Flavored Markdown for the section I edited. Github supports markdown natively. And this is much easier to read! Let me know if this is ok - I think it'll make maintenance and community contributions easier.

Thanks for all your work!
